spring-web/dependencies.gradle

```diff
@@ -0,0 +1,20 @@
+dependencies {
+    implementation project(":core")
+
+    // Spring
+    compileOnly 'org.springframework.boot:spring-boot-starter-web'
+
+    // LSD
+    api 'io.github.lsd-consulting:lsd-core:5.6.0'
+    implementation 'io.github.lsd-consulting:lsd-formatting-library:1.0.8'
+    compileOnly 'io.github.lsd-consulting:lsd-distributed-connector:2.0.0'
```

Comment:
noticed some lsd libs can be upgraded but not coupled to this PR so we can come back later